tests/foreman/installer/test_installer.py

```diff
@@ -1299,21 +1300,24 @@ def test_positive_foreman_module(target_sat):
 
     :CaseLevel: System
 
-    :expectedresults: Foreman RPM and SELinux module versions match
+    :expectedresults: Foreman RPM and SELinux module are both present on the satellite
     """
     rpm_result = target_sat.execute('rpm -q foreman-selinux')
     assert rpm_result.status == 0
 
     semodule_result = target_sat.execute('semodule -l | grep foreman')
     assert semodule_result.status == 0
 
-    # Sample rpm output: foreman-selinux-1.7.2.8-1.el7sat.noarch
-    version_regex = re.compile(r'((\d\.?)+[-.]\d)')
-    rpm_version = version_regex.search(rpm_result.stdout).group(1)
-    # Sample semodule output: foreman        1.7.2.8
-    semodule_version = version_regex.search(semodule_result.stdout).group(1)
-    rpm_version = rpm_version[:-2]
-    assert rpm_version.replace('-', '.') == semodule_version
+    # Only check module version on RHEL 7 and below
+    # since it was removed from semodule in RHEL 8
+    if settings.server.version.rhel_version <= 7:
```

Comment:
```suggestion
    if target_sat.os_version.major <= 7:
```

Reply:
I was wondering what the best way to access the RHEL version was. I like your suggestion much more. Pushing up now.